openfisca_country_template/variables/housing.py

```diff
@@ -24,3 +24,19 @@ class rent(Variable):
     entity = Household
     definition_period = MONTH
     label = u"Rent paid by the household"
+
+
+HOUSING_STATUS = Enum([
+    u'Tenant',
+    u'Owner',
+    u'Free logder',
+    u'Homeless'])
+
+
+class housing_occupancy_status(Variable):
+    column = EnumCol(
+        enum = HOUSING_STATUS,
```

Comment:
The `,` is non necessary (though inoffensive), I removed it.
I defined the enum outside to be able to import it from `taxes.py`, so that I can do:
```py
tenant = (occupancy_status == HOUSING_OCCUPANCY_STATUS['Tenant'])
```
instead of
```
tenant = (occupancy_status == 0)
```